ext/sst/INSTALL.md

```diff
@@ -11,14 +11,14 @@ installed.
 ### Downloading the SST-Core Source Code
 
 ```sh
-wget https://github.com/sstsimulator/sst-core/releases/download/v11.1.0_Final/sstcore-11.1.0.tar.gz
-tar xf sstcore-11.1.0.tar.gz
+wget https://github.com/sstsimulator/sst-core/releases/download/v13.0.0_Final/sstcore-13.0.0.tar.gz
+tar xvf sstcore-13.0.0.tar.gz
```

Comment:
```suggestion
tar xzf sstcore-13.0.0.tar.gz
```

Minor nits here:

1. `v` is just `verbose`. This is an uncesssary addition/
2. I like to be explicit and add `z` which means to also unzip (though it does do this automatically).

Reply:
Sure, I'll update this.